docs/orleans/tutorials-and-samples/snippets/minimal/Client/Program.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Orleans.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using GrainInterfaces;
+
+try
+{
+    using var host = await StartClientAsync();
+    var client = host.Services.GetRequiredService<IClusterClient>();
+
+    await DoClientWorkAsync(client);
+    Console.ReadKey();
+
+    return 0;
+}
+catch (Exception e)
+{
+    Console.WriteLine($$"""
+        Exception while trying to run client: {{e.Message}}
+        Make sure the silo the client is trying to connect to is running.
+        Press any key to exit.
+        """);
+    
+    Console.ReadKey();
+    return 1;
+}
+
+static async Task<IHost> StartClientAsync()
+{
+    var builder = new HostBuilder()
+        .UseOrleansClient(client =>
+        {
+            client.UseLocalhostClustering()
+                .Configure<ClusterOptions>(options =>
+                {
+                    options.ClusterId = "dev";
```

Comment:
The ClusterOptions config shouldn't be needed in 7.0: it will default to `"default"`